packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -449,11 +450,17 @@ class AttachDisk extends Component {
 }))
 @connectStore(() => {
   const getAllVbds = createGetObjectsOfType('VBD')
+  const getIsoSrs = createGetObjectsOfType('SR').filter(
+    (_, { pool: { $poolId } }) =>
+      sr =>
+        sr.$poolId === $poolId && sr.SR_type === 'iso'
```

Comment:
`$poolId` is deprecated. Use `$pool` or even `id` instead.
https://github.com/vatesfr/xen-orchestra/blob/ab3f26e2dab46d508df1a5012e7f50a2a1a65f92/packages/xo-server/src/xapi-object-to-xo.mjs#L874